cloudwatch-logs-s3-forwarder/variables.tf

```diff
@@ -0,0 +1,48 @@
+variable "name_prefix" {
+  description = "A prefix that will be added to resources so that they are unique"
```

Comment:
Nitpicking but some descriptions ends with a period and some don't.

Reply:
I'll fix it. Nitpicking is good 👍